chainer/functions/pooling/unpooling_2d.py

```diff
@@ -0,0 +1,68 @@
+import numpy
+
+from chainer import cuda
+from chainer.functions.pooling import pooling_2d
+from chainer.utils import conv
+from chainer.utils import type_check
+
+
+class Unpooling2D(pooling_2d.Pooling2D):
+
+    """Unpooling over a set of 2d planes."""
+
+    def __init__(self, ksize, stride=None, pad=0,
+                 outsize=None, cover_all=True):
+        super(Unpooling2D, self).__init__(ksize, stride, pad, cover_all)
+        self.outh, self.outw = (None, None) if outsize is None else outsize
```

Comment:
By setting `outh` and `outw` as attributes, I think we assume that the height of expected outsize is same throughout all training data. Do you think it is reasonable one?